### PR TITLE
react(tianmu): refactor some code about tianmu engine(#11)

### PR DESCRIPTION
**Pull request by RingsC**
This is one series of #11, which is used to make tianmu engine more readable.

Issue Number: close #11

## Tests Check List

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

## Changelog

- [ ] New Feature
- [ ] Bug Fix
- [x] Improvement
- [ ] Performance Improvement
- [ ] Build/Testing/CI/CD
- [ ] Documentation
- [ ] Not for changelog (changelog entry is not required)

## Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features

### CODE REVIEW

**Review by adofsauron — APPROVED** [review]
LGTM

File: storage/tianmu/handler/tianmu_handler_com.cpp [hunk 1]
```diff
@@ -218,8 +218,7 @@ int rcbase_init_func(void *p) {
     if (hent) strmov_str(global_hostIP_, inet_ntoa(*(struct in_addr *)(hent->h_addr_list[0])));
     my_snprintf(global_serverinfo_, sizeof(global_serverinfo_), "\tServerIp:%s\tServerHostName:%s\tServerPort:%d",
                 global_hostIP_, glob_hostname, mysqld_port);
-    ha_kvstore_ = new index::KVStore();
-    ha_kvstore_->Init();
+    //starup tianmu engine.
```
Comment (andyli029) [inline storage/tianmu/handler/tianmu_handler_com.cpp]: starup 
->
startup?

Comment (RingsC) [reply storage/tianmu/handler/tianmu_handler_com.cpp:221]: OK.

Comment (RingsC) [reply storage/tianmu/handler/tianmu_handler_com.cpp]: Done.

**Review by andyli029 — APPROVED** [review]
LGTM.

**Pull request merged**